modules/admin_manual/pages/release_notes.adoc

```diff
@@ -30,6 +31,130 @@
+
+=== New Background Job for Change Detection in Federated Shares
+
+With ownCloud Server 10.2.1 a https://doc.owncloud.com/server/admin_manual/release_notes.html#background-job-for-change-detection-of-nested-federated-shares[background job for change detection of nested federated shares] was added (`occ incoming-shares:poll`) to allow ownCloud Server to discover changes in federated shares in order to make them available for synchronization with the ownCloud Clients. Based on feedback a new, improved background job with more configuration options was added to Server 10.5. It replaces the former occ command which is now **deprecated** and should not be used anymore after upgrading to 10.5.
+
+In addition to discovering changes ("check"), the new background job also synchronizes meta data changes between involved servers ("scan") making them available without requiring users to actively browse them. 
+
+The new background job provides some configuration options to optimize its performance, especially for larger environments:
+- Minimum amount of hours since the last login of a user that a scan is triggered (limits the execution of discovery & meta data sync to active users which have logged in during the configured time frame) (default: 24h)
+
+`occ config:app:set files_sharing cronjob_scan_external_min_login --value <integer-seconds>`
+  
+- Minimum amount of hours since the last scan of a federated share for the next scan to be triggered (avoids frequently scanning the same federated share when it is in active use) (default: 3h)
+
+`occ config:app:set files_sharing cronjob_scan_external_min_scan --value <integer-seconds>`
+
+- Maximum amount of federated shares scanned per execution (scan is only performed if changes in federated shares are discovered) (default: 100)
+
+`occ config:app:set files_sharing cronjob_scan_external_batch --value <integer-number>`
+
+The new background job is disabled by default and can be enabled/disabled in the _Admin -> Sharing_ settings (_Periodically synchronize outdated federated shares for active users_) or using `occ config:app:set files_sharing cronjob_scan_external_enabled --value yes`. If enabled, it will be executed as part of the regular ownCloud background job queue and therefore does not need to be added to crontab.
+
+TIP: If your instance provides federated sharing, activate the new background job as explained above.
+TIP: Remove the `occ incoming-shares:poll` command from crontab if you have set it.
+
+=== Other Notable Changes
+
+- The "Add to your ownCloud" feature on public link pages is now hidden if an instance has outgoing federated shares disabled (specifically, if _Allow users on this server to send shares to other servers_ is disabled in Admin -> Sharing settings) https://github.com/owncloud/core/pull/37232[#37232]
```

Comment:
`disabled in Admin -> Sharing settings` --> `menu:Settings[Admin > Sharing] settings`